webserver/external/messybrainz.py

```diff
@@ -0,0 +1,36 @@
+import requests
+import ujson
+
+def get_messybrainz_id(querydata):
+    """
+    This function queries messybrainz to get a msid according to the 'artist/title' present in the item metadata
+    Args:
+        querydata: A dictionary containing at least artis and song title
+
+    Returns:
+        msid: string messybrainz id -> if it is found in the messybrainz database
+        None: if no reference is fount in messybrainz database
+
+    """
+    msurl = 'https://messybrainz.org/submit'
```

Comment:
This should be a config parameter in `config.py`, so that people can run their own messybrainz server during development. We don't want to submit bad data to messybrainz when we are working on new features.